certified-connectors/Azure Machine Learning/apiProperties.json

```diff
@@ -0,0 +1,45 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aadcertificate",
+          "clientId": "4e68efd7-d87d-45cb-afc4-7438a8b8d159",
+          "scopes": [],
+          "redirectMode": "global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://management.azure.com/"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFE",
```

Comment:
I actually have provided an icon with transparent background. The icon itself is blue in color so I decided to have a color as close to white as possible (since #FFFFFF is not a valid value here)